map: 1

sensor_poses:
  - [5.5,  -14.0, 0.5, 0.0, 0.0, 0.0] # 1 (top - right)
  - [5.5,  -10.6, 0.5, 0.0, 0.0, 0.0] # 2 (top)
  - [5.5,   -7.3, 0.5, 0.0, 0.0, 0.0] # 3
  - [5.5,   -4.2, 0.5, 0.0, 0.0, 0.0] # 4
  - [4.5,    0.7, 0.5, 0.0, 0.0, 0.0] # 5
  - [4.0,    5.4, 0.5, 0.0, 0.0, 0.0] # 6
  - [4.5,    8.4, 0.5, 0.0, 0.0, 0.0] # 7
  - [4.5,   12.0, 0.5, 0.0, 0.0, 0.0] # 8 (top - left)
  - [-4.4,  12.6, 0.5, 0.0, 0.0, 0.0] # 9 (bottom - left)
  - [-4.4,   9.6, 0.5, 0.0, 0.0, 0.0] # 10
  - [-4.4,   6.7, 0.5, 0.0, 0.0, 0.0] # 11
  - [-4.4,   3.0, 0.5, 0.0, 0.0, 0.0] # 12
  - [-4.4,  -0.6, 0.5, 0.0, 0.0, 0.0] # 13
  - [-4.4,  -4.2, 0.5, 0.0, 0.0, 0.0] # 14
  - [-4.4,  -7.7, 0.5, 0.0, 0.0, 0.0] # 15
  - [-4.4, -11.0, 0.5, 0.0, 0.0, 0.0]
  - [-4.4, -13.1, 0.5, 0.0, 0.0, 0.0] # 17 (bottom - right)

sampling:
  distribution: 1
  seed: 42
  radius_min: 0.0
  radius_inc: 0.1
  radius_steps: 11
  poses: 100
optimizer:
  iterations: 50
  dist_converged: 0.01

